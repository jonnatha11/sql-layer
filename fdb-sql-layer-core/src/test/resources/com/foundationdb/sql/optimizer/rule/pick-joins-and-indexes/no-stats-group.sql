SELECT name, order_date, quan, city
  FROM customers, orders, items, addresses
 WHERE customers.cid = orders.cid
   AND orders.oid = items.oid
   AND customers.cid = addresses.cid
   AND items.sku = '1234'