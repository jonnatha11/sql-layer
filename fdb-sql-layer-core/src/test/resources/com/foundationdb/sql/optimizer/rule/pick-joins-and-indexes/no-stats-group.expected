SelectQuery@4df8b14
  ResultSet@7023d08b[name, order_date, quan, city]
    Project@3f508f31[customers.name, orders.order_date, items.quan, addresses.city]
      Select@41e8fa70[]
        TableGroupJoinTree@2691f0de(TableGroup@36e3ffdf(customers), customers INNER addresses INNER orders INNER items - SingleIndexScan@2c3299f6(Index(test.items.sku[IndexColumn(sku)]), NONE, =1234, rows = 1, cost = 302.452))
          TableSource@59db9f45(customers - TableGroup@36e3ffdf(customers))
            TableSource@707efa96(addresses - TableGroupJoin@4fef3a22(Join(test/customers/cid/test/test.addresses/cid: test.addresses -> test.customers, group(test.customers))))
            TableSource@61b59919(orders - TableGroupJoin@62d1b72d(Join(test/customers/cid/test/test.orders/cid: test.orders -> test.customers, group(test.customers))))
              TableSource@38650646(items - TableGroupJoin@36511e0a(Join(test/orders/oid/test/test.items/oid: test.items -> test.orders, group(test.customers))))
